azurerm/internal/services/sentinel/tests/resource_arm_sentinel_alert_rule_action_test.go

```diff
@@ -0,0 +1,191 @@
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "azurerm_sentinel_alert_rule_action" {
+			continue
+		}
+
+		id, err := parse.SentinelAlertRuleActionID(rs.Primary.ID)
+		if err != nil {
+			return err
+		}
+
+		if resp, err := client.GetAction(ctx, id.ResourceGroup, id.Workspace, id.Rule, id.Name); err != nil {
+			if !utils.ResponseWasNotFound(resp.Response) {
+				return fmt.Errorf("Getting on Sentinel.AlertRules: %+v", err)
+			}
+		}
+
+		return nil
+	}
+
+	return nil
+}
+
+func testAccAzureRMSentinelAlertRuleAction_basic(data acceptance.TestData) string {
+	template := testAccAzureRMSentinelAlertRuleAction_template(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_sentinel_alert_rule_action" "test" {
+  name         = "acctest-AlertRuleAction-%d"
+  rule_id      = azurerm_sentinel_alert_rule_scheduled.test.id
+  logic_app_id = azurerm_logic_app_workflow.test.id
+  depends_on   = [azurerm_logic_app_trigger_custom.test]
+}
+`, template, data.RandomInteger)
+}
+
+func testAccAzureRMSentinelAlertRuleAction_requiresImport(data acceptance.TestData) string {
+	template := testAccAzureRMSentinelAlertRuleAction_basic(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_sentinel_alert_rule_action" "import" {
+  name         = azurerm_sentinel_alert_rule_action.test.name
+  rule_id      = azurerm_sentinel_alert_rule_action.test.rule_id
+  logic_app_id = azurerm_sentinel_alert_rule_action.test.logic_app_id
+}
+`, template)
+}
+
+func testAccAzureRMSentinelAlertRuleAction_template(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-sentinel-%[1]d"
+  location = "west europe"
+}
+
+resource "azurerm_log_analytics_workspace" "test" {
+  name                = "acctest-workspace-%[1]d"
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
+  sku                 = "pergb2018"
+}
+
+resource "azurerm_sentinel_alert_rule_scheduled" "test" {
+  name                       = "acctest-SentinelAlertRule-Sche-%[1]d"
+  log_analytics_workspace_id = azurerm_log_analytics_workspace.test.id
+  display_name               = "Some Rule"
+  severity                   = "High"
+  query                      = <<QUERY
+AzureActivity |
+  where OperationName == "Create or Update Virtual Machine" or OperationName =="Create Deployment" |
+  where ActivityStatus == "Succeeded" |
+  make-series dcount(ResourceId) default=0 on EventSubmissionTimestamp in range(ago(7d), now(), 1d) by Caller
+QUERY
+}
+
+resource "azurerm_logic_app_workflow" "test" {
+  name                = "acctest-LogicApp-%[1]d"
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
+}
+
+resource "azurerm_logic_app_trigger_custom" "test" {
+  name         = "When_a_response_to_an_Azure_Sentinel_alert_is_triggered"
```

Comment:
if this is a constant - is it worth making a new logic app trigger for `azurerm_logic_app_trigger_sentinel_alert`?